src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -6,82 +6,82 @@
-    	}
-    	catch(MalformedURLException err)
-    	{
-    		getLogger().error("Unable to initialize " + this.getClass().getSimpleName() + " module", err);
-    	}
+        String stacksURL = appInstance.getProperties().getPropertyStr("stacksURL", defaultUrl);
+        try
+        {
+            baseAuthUrl = new URL(stacksURL);
+            getLogger().info("Initialized " + this.getClass().getSimpleName() + " at " + baseAuthUrl);
+        }
+        catch (MalformedURLException err)
+        {
+            getLogger().error("Unable to initialize " + this.getClass().getSimpleName() + " module", err);
+        }
     }
 
-    
+    // Invoked when an HTTP MPEGDash Streaming session is created;
+    // defined in IModuleOnHTTPMPEGDashStreamingSession module interfaces
     public void onHTTPMPEGDashStreamingSessionCreate(HTTPStreamerSessionMPEGDash httpSession)
     {
-    	getLogger().info("Tommy: SulWowza onHTTPMPEGDashStreamingSessionCreate: " + httpSession.getSessionId());
-    	authorizeSession(httpSession);
+        getLogger().info("Tommy: SulWowza onHTTPMPEGDashStreamingSessionCreate: " + httpSession.getSessionId());
+        authorizeSession(httpSession);
     }
-    
+
+    // Invoked when an HTTP Cupertino Streaming session is created;
+    // defined in IModuleOnHTTPCupertinoStreamingSession interface
     public void onHTTPCupertinoStreamingSessionCreate(HTTPStreamerSessionCupertino httpSession)
     {
-		getLogger().info("Tommy: SulWowza onHTTPCupertinoStreamingSessionCreate: ");
-		authorizeSession(httpSession);
-	}
-    
+        getLogger().info("Tommy: SulWowza onHTTPCupertinoStreamingSessionCreate: ");
+        authorizeSession(httpSession);
+    }
+
     void authorizeSession(IHTTPStreamerSession httpSession)
     {
-    	String query = httpSession.getQueryStr();
-    	if(authorize(query))
-    		httpSession.acceptSession();
-    	else httpSession.rejectSession();
+        String queryStr = httpSession.getQueryStr();
+        if (authorize(queryStr))
+            httpSession.acceptSession();
+        else
+            httpSession.rejectSession();
     }
-    
-    boolean authorize(String query)
+
+    boolean authorize(String queryStr)
     {
-    	String authToken = getAuthToken(query);
-    	return validateAuthToken(authToken);
+        String authToken = getAuthToken(queryStr);
+        return validateAuthToken(authToken);
     }
-    
-    String getAuthToken(String parameters)
+
+    String getAuthToken(String queryStr)
     {
-    	if (parameters != null)
-    	{
-    		String[] parts = parameters.split("\\?");
-    		String query = parts[parts.length-1];
-    		List<NameValuePair> httpParams = URLEncodedUtils.parse(query,Charset.defaultCharset());
-    		for (NameValuePair param:httpParams)
-    			if (param.getName().equals("token"))
-    				return param.getValue();
-    	}
-    	return null;
+        if (queryStr != null && queryStr.length() > 6) // "token=" is 6 chars
```

Comment:
Do you think a constant would be better or worse for inebriated coder test?  I'm inclined to leave it as is, especially because I can't think of a good name for the constant.  TOKEN_LENGTH makes me think of the length of the token, not the param name.   Is MIN_TOKEN_PARAM_LENGTH more clear than the code as is?

Using 6 with a comment, or using a constant, is definitely more efficient java code than instantiating a String and computing a length every time.   Or did you mean that would be the value of the constant?  String.new("token=").length()  ?